epictrack-web/src/services/projectService/projectService.ts

```diff
@@ -5,8 +5,11 @@ import { MasterBase } from "../../models/type";
 import { ListType } from "../../models/code";
 
 class ProjectService implements ServiceBase {
-  async getAll(return_type?: string) {
-    return await http.GetRequest(Endpoints.Projects.PROJECTS, { return_type });
+  async getAll(return_type?: string, with_works?: boolean) {

```

Comment:
with_works? is includeWorks better?

Reply:
i am not sure you want project which has a work or you wanna include a work ..so plz feel to disregard this

Reply:
return_type ? is it the project type?

Reply:
I need only the project with works, I don't need to include the works info.

Reply:
return_type  is not my addition, it's already there. It decides what schema to use